src/Features/CSharp/Portable/StringIndentation/CSharpStringIndentationService.cs

```diff
@@ -31,6 +31,11 @@ public CSharpStringIndentationService()
             Document document, TextSpan textSpan, CancellationToken cancellationToken)
         {
```

Comment:
i'm genuinely curious how there could be an issue here versus, say, classification (which also walks the tree).  Is it jsut because this is in-proc?  perhaps we can just move this OOP to then be able to reuse costs in VS and share teh costs in the other process.